.github/dependabot.yml

```diff
@@ -0,0 +1,22 @@
+version: 2
+updates:
+  - package-ecosystem: "pip"
+    directory: "/"
+    schedule:
+      interval: "daily"
+      time: "07:00"
+    versioning-strategy: auto
+    # Allow up to 10 open pull requests for updates to dependency versions
+    open-pull-requests-limit: 10
+    labels:
+      - "part:tooling"
+    reviewers:
+      - "frequenz-floss/python-sdk-team"
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    labels:
+      - "part:tooling"
```

Comment:
Can you add `type:tech-debt` too (for both)?

Reply:
sure, I was not sure about which other labels to add

Reply:
I think we are good with these 2. There is also prio, but I guess for now it is fine to leave it without, unless is a security update, if it is maybe `prio:high` would be a good.

Reply:
I'm afraid we can only set `priority:high` for security updates if we define a dedicated update for the default branch as it was done for the supported branches.

Reply:
I see, and do you know if there is no way to use wildcards for `target-branch` then? I had a quick look at the docs and it doesn't say, I wonder if you tried it.

Also the docs says that `target-branch` doesn't apply to security updates, so I'm not sure how this will play along.

BTW, using [YAML anchors](https://community.atlassian.com/t5/Bitbucket-questions/YAML-anchors-and-variables-code-reuse-question/qaq-p/1733738) we can remove **a lot** of duplication. But anyway, if we are going to need to keep this file updated with each release, I think it will get out of sync pretty soon, so I'm not sure it is worth adding the `target-branch` if there is no wildcard support (at least until we include updating this file in the release automation).

Had a more in depth look and looking at the dependabot code, no, it doesn't support wildcards, [is doing just a simple comparison](https://github.com/dependabot/dependabot-core/blob/e64b21e4304b98570a585b8ed6fd74cd90142cfb/common/lib/dependabot/config/file.rb#L21).

I looked for issues and only found one about [supporting arrays for `target-branch`](https://github.com/dependabot/dependabot-core/issues/2511), but that doesn't help us, so [I created one for wildcards support](https://github.com/dependabot/dependabot-core/issues/6890).

Reply:
I'd say let's ignore other branches for now until things are more stable, we have better automation, or the feature is added. At the most we could add a `stable` branch to point to the current release and add a dependabot config only for that one. But I would skip that for now too, let's first try this out.